ietf/doc/utils.py

```diff
@@ -1024,6 +1024,7 @@ def build_file_urls(doc: Union[Document, DocHistory]):
             if doc.tags.filter(slug="verified-errata").exists():
                 file_urls.append(("with errata", settings.RFC_EDITOR_INLINE_ERRATA_URL.format(rfc_number=doc.rfc_number())))
         file_urls.append(("bibtex", urlreverse('ietf.doc.views_doc.document_bibtex',kwargs=dict(name=name))))
+        file_urls.append(("bibxml", urlreverse('ietf.doc.views_doc.document_bibxml',kwargs=dict(name=name))))
```

Comment:
This isn't what we want - the bibxml3 endpoint will (and should) 404 on an rfc name. Please remove just this line.

It's not perfect since then there will be no way to get the bibxml for something that has become an rfc. The solution that would send someone off to bib.ietf.org would be better at this.